jujugui/static/gui/src/app/components/profile/profile.js

```diff
@@ -45,8 +46,20 @@ class Profile extends React.Component {
 
   render() {
     const props = this.props;
+    const profileUrl = this._getProfileURL();
     const isActiveUsersProfile = props.controllerUser.split('@')[0] === props.userInfo.profile;
     const sectionsMap = new Map();
+
+    console.log(profileUrl.full);
+
+    if (profileUrl.full === 'revenue-statement') {
```

Comment:
If we ever change the url scheme for the profile then this will fail. in the return value for `_getProfileURL()` it should return a getter `isRevenueStatement()` or something which you could call to check. Also then the method name and description should probably be updated.